### PR TITLE
Update e2e DRA README.md

### PR DESCRIPTION
**Pull request by byako**
Update e2e DRA README.md

### CODE REVIEW

File: test/e2e/dra/test-driver/README.md [hunk 1]
```diff
@@ -48,9 +48,14 @@ kubelet<->dynamic resource allocation plugin interaction.
...
 
+etcd is required to be in your PATH, install it by calling
+```console
+./hack/install_etcd.sh
+```
+
 To try out the feature, build Kubernetes, then in one console run:
 ```console
-FEATURE_GATES=DynamicResourceAllocation=true KUBELET_RESOLV_CONF="/etc/resolv-9999.conf" DNS_ADDON="coredns" CGROUP_DRIVER=systemd CONTAINER_RUNTIME_ENDPOINT=unix:///var/run/crio/crio.sock LOG_LEVEL=6 ENABLE_CSI_SNAPSHOTTER=false API_SECURE_PORT=6444 ALLOW_PRIVILEGED=1 PATH=/nvme/gopath/src/k8s.io/kubernetes/third_party/etcd:$PATH ./hack/local-up-cluster.sh -O
+FEATURE_GATES=DynamicResourceAllocation=true KUBELET_RESOLV_CONF="/var/run/systemd/resolve/resolv.conf" CGROUP_DRIVER=systemd CONTAINER_RUNTIME_ENDPOINT=unix:///var/run/crio/crio.sock LOG_LEVEL=6 API_SECURE_PORT=6444 ALLOW_PRIVILEGED=1 PATH="$(pwd)/third_party/etcd:$PATH" ./hack/local-up-cluster.sh -O
```
Comment (bart0sh) [inline test/e2e/dra/test-driver/README.md:58]: I dont' have /var/run/systemd/resolve directory on my system, so this wouldn't work for me.
I'm not using KUBELET_RESOLV_CONF either as my /etc/resolv.conf points to the real name servers.

Comment (byako) [reply test/e2e/dra/test-driver/README.md:58]: Did you disable systemd-resolved manually, or does Fedora not have it? I'll check in latest Fedora VM if it comes by default

**Comment by bart0sh** [comment]
I don't remember, but it could be the case. Not sure about Fedora either. I'm using Ubuntu 18.04 LTS.

File: test/e2e/dra/test-driver/README.md (hunk 1 above)
Comment (bart0sh) [reply test/e2e/dra/test-driver/README.md:58]: I don't remember, but it could be the case. Not sure about Fedora either. I'm using Ubuntu 18.04 LTS.

Comment (byako) [reply test/e2e/dra/test-driver/README.md:58]: Fedora 36, Ubuntu 20.04, Ubuntu 22.04 seem to have that by default.

Comment (bart0sh) [reply test/e2e/dra/test-driver/README.md:58]: I'd suggest to leave only env variables that are mandatory for DRA and emphasise this somehow. We can give a real example with KUBELET_RESOLV_CONF and other non-mandatory options though.

Comment (pohly) [reply test/e2e/dra/test-driver/README.md:58]: I ended up simplifying the invocation in https://github.com/pohly/kubernetes/pull/20 before I noticed this PR.

**Pull request closed without merging**